Description: Do not use --quiet options to get the correct exit code
Author: Andriy Senkovych <jolly_roger@itblog.org.ua>
Origin: other, https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=758432
Bug-Debian: http://bugs.debian.org/758432
Forwarded: https://github.com/buildbot/buildbot/pull/1267
Date: Sun, 19 Oct 2014 22:16:38 +0300

Index: debian-buildbot/contrib/init-scripts/buildmaster.init.sh
===================================================================
--- debian-buildbot.orig/contrib/init-scripts/buildmaster.init.sh
+++ debian-buildbot/contrib/init-scripts/buildmaster.init.sh
@@ -108,7 +108,7 @@ function master_op () {
 
     ${MASTER_PREFIXCMD[$mi]} \
     su -s /bin/sh \
-    -c "$MASTER_RUNNER $op --quiet ${MASTER_OPTIONS[$mi]} ${MASTER_BASEDIR[$mi]}" \
+    -c "$MASTER_RUNNER $op ${MASTER_OPTIONS[$mi]} ${MASTER_BASEDIR[$mi]} > /dev/null" \
     - ${MASTER_USER[$mi]}
     return $?
 }
